app/views/unattended/provisioning_templates/PXELinux/preseed_default_pxelinux_autoinstall.erb

```diff
@@ -34,6 +34,7 @@ test_on:
   end
   options << "locale=#{host_param('lang') || 'en_US'}"
   options = options.join(' ')
+  image_path = @preseed_path.sub(/\/$/, '.iso')
```

Comment:
I have expected the value of the `url` parameter to change.
But then I realized you want to replace the "last slash and line-end" with "dot iso", but in the example here we don't have a slash.

```suggestion
  image_path = @preseed_path.sub(/\/?$/, '.iso')
```

Reply:
That's the last change I request, promise :)

Reply:
No worries & thanks for all your enhancements on this! :)